hadoop-ozone/common/pom.xml

```diff
@@ -111,6 +111,10 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-hdfs-client</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.ozone</groupId>
+      <artifactId>hdds-server-framework</artifactId>
```

Comment:
Please avoid this.

`hdds-server-framework` is for server-only shared code.  `ozone-common` is shared between client and server.  Client should not be getting server-side code and dependencies.

Reply:
Noted, I was doing this since `OzoneManagerProtocol.java`, `OzoneManagerProtocolClientSideTranslatorPB.java` required `ManagedSecretKey` from the package `org.apache.hadoop.hdds.security.symmetric` - I will take a look at this to see how this can be avoided, thanks.

Reply:
Done, I have made changes to avoid importing this particular dependency.